packages/client/src/client.js

```diff
@@ -239,6 +248,7 @@ export class PercyClient {
 
         // not finished, poll again
         if (pending) {
+          setTimeout(self.log.debug, interval, 'waiting...');
```

Comment:
Why do we need setTimeout again? Isn't this entire function being called after `interval`? In that case we can just use `log.debug(...)`

Reply:
@yashdsaraf I just forgot to mention this, we're using `setTimeout` here because only then its possible to show
 `waiting... (10000ms)`

we can't use it directly otherwise it will show something like

```
waiting... (1ms)
Get Build 1234 (10100ms)
```

Reply:
removed this log line for now